src/backend/chat_service/chat_service.py

```diff
@@ -149,4 +172,4 @@ async def check_server_status(request: Request):
 if __name__ == "__main__":
     import uvicorn
 
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Comment:
Looks like the same line is added with spaces. Make sure this change is not present in this PR